### PR TITLE
No unknown params for adjust SDK

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

File: src/adjust/adjustfiltering.cpp [hunk 1]
```diff
@@ -146,7 +160,6 @@ QUrlQuery AdjustFiltering::filterParameters(QUrlQuery& parameters,
       continue;
     }
 
-    newParameters.addQueryItem(parameter.first, parameter.second);
```
Comment (mbirghan) [inline src/adjust/adjustfiltering.cpp:149]: Do we just want to remove them completely or replace them with dummy values?

**Review by mbirghan — CHANGES_REQUESTED** [review]
Just one question otherwise awesome!

File: src/adjust/adjustfiltering.cpp (hunk 1 above)
Comment (bakulf) [reply src/adjust/adjustfiltering.cpp:149]: we do not have dummy values... Better to remove them than sending invalid values.

Comment (mbirghan) [reply src/adjust/adjustfiltering.cpp:149]: Ok sounds good!

**Review by mbirghan — APPROVED** [review]

**Pull request merged**